tools/kafka/loopback/README.md

```diff
@@ -0,0 +1,50 @@
+# 100K Loopback
+
+This service is a helper for 100k-epic, transfers incoming payload from a **device-data** topic to **device-manager** topic in the dojot style.
```

Comment:
My suggestion is to replace the text by something like:

This service is a helper for testing the 100k-epic. It's goal is to transform messages sent from devices to dojot into messages sent from dojot to devices. To implement this "loopback", it transfers incoming messages from **device-data** topic to **device-manager** topic adapting them according to dojot's message schema.